ui/src/app/components/runs/filters/string-filter/string-filter.component.spec.ts

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
Please add some tests. E.g. you could subscribe to the `changes` subject and call `modelChanged` or emit values to `removeFilterSubject`

Reply:
`accepts` also needs to be tested.

Reply:
accepts has been tested